docs/source/user/input_file_overview.rst

```diff
@@ -0,0 +1,140 @@
+.. _input_file_overview:
+
+
+Overview of input file formats
+==============================
+
+OpenFAST uses two primary input file formats: *value column* where the first
+value on the line is read, and *key+value* where a value and keyword pair are
+read. Both formats are line number based where a specific input is expected on a
+specific line, with some exceptions. 
+
+.. _sec_value_column:
+
+Value column input files
+------------------------
+
+Only the first column in a *value column* based input file is read.  This is the
+historical format used by OpenFAST and it's predecessors. Everything after the
+first value read is simply ignored by the code.  This allowed the user to keep
```

Comment:
Add something like: "(the keyword was often referenced in the source code and documentation, but OpenFAST did not process the keyword or description)" here.